City-of-Helsinki/helsinki-paatokset
# PP-231: Decisions data aggregation

This PR adds aggregation for decisions, adds fixes to data aggregation scripts and updates static data files. The cases and meetings migrations are also updated to fetch existing node IDs, so different migration variants do not cause duplicates.

** To test **
- Checkout branch, SSH into the container and run `drush cr; drush cim -y`
- Run `drush ap:fs -v`
- Check https://helsinki-paatokset.docker.so/fi/ahjo-proxy/aggregated/decisions_latest and https://helsinki-paatokset.docker.so/fi/ahjo-proxy/aggregated/decisions_all
  - There should be 1739 items in all and 438 items in latest
- Run `drush migrate-status --tag=ahjo_api`
- There should be 4179 items in all meetings and 52 items in latest meetings
- Run `drush migrate-import ahjo_meetings:latest`
- Run `drush migrate-import ahjo_meetings:all`
  - Run `drush sql-cli` and `select count(id) from node where type='meeting';`
  - There should be only be around 4179 meeting nodes in total (not 4179 + 52 for example).
